vault/share/plugins/vault/30_generic_methods/variable_from_vault.cf

```diff
@@ -62,7 +62,7 @@ bundle agent variable_from_vault(variable_prefix, variable_name, path)
         ifvarclass => not(isvariable("vault.auth_token"));
 
     pass3.variable_defined::
-      "repair" usebundle => _classes_repaired("${old_class_prefix}");
+      "success" usebundle => _classes_success("${old_class_prefix}");
       "repair" usebundle => _classes_repaired("${class_prefix}");
```

Comment:
We also need to replace the class defined for `class_prefix`.

Reply:
fixed it up!